### PR TITLE
Ser 218 soil training correction

### PR DESCRIPTION
**Pull request by Bahatiroben**
Soil fertility training. Most of the reviews will be code reviews since functionality needs a physical phone with a kenyan sim.
review through the commits and [telerivet's actions](https://telerivet.com/p/0c6396c9/service/SV77cf8fe0e28a3bf3)
test from here: https://telerivet.com/p/0c6396c9/service/SV7684c662f118f82e/edit
- dial in the short code
- enter the account number: 24450523  
- choose trainings (option 4 on clients menu)
- choose soil trainings (option 11 on the second screen)

this will trigger a 2 way sms service and
you should see a notice that you have received an sms on your phone for trainings.

A 2 way sms service will be triggered upon any response to any message, there will be a next message that matches the flow in [this user manual](https://docs.google.com/spreadsheets/d/1sV3nQ57RRJ9oR8kOGDgH3VRR5BlemRIKxVySvHwL548/edit#gid=617330296).

2-WAY SMS testing (https://telerivet.com/p/0c6396c9/service/SV77cf8fe0e28a3bf3/edit)

go to the above link on the 2 way sms for soil training
- change the route to 21234 instead of the usual shortcode
-click Simulate service.
- there should be a series of messages.
- Once a batch of messages is sent and its not prompting to answer any question, you can reply with any thing to get the next batch.
- There is one prompt to which you can respond with a or b.
- keep responding with anything until all the messages are sent.

- NB: all messages will be sent in the exact order that they were given here(https://docs.google.com/spreadsheets/d/1sV3nQ57RRJ9oR8kOGDgH3VRR5BlemRIKxVySvHwL548/edit#gid=617330296)

### CODE REVIEW

File: ke-legacy/main.js [hunk 1]
```diff
@@ -34,17 +34,7 @@ var warrantyExpiration = require('../warranty-expiration/warrantyExpiration');
...
-    defaultEnvironment = 'prod';
-}else{
-    defaultEnvironment = 'dev';
-}
 
-if(service.vars.env === 'prod' || service.vars.env === 'dev'){
-    env = service.vars.env;
-}else{
-    env = defaultEnvironment;
-}
```
Comment (DevFojo) [inline ke-legacy/main.js:47]: Why remove this block?

Comment (Bahatiroben) [reply ke-legacy/main.js:47]: @DevFojo it was a duplicate. its already implemented on line 1-13

File: soil-fertility-trainings/responseHandlers/batch2ResponseHandler.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            var getMessage = translator(translations, lang);
+            var messages = ['sms-3.1', 'sms-3.2', 'sms-3.3', 'sms-3.4', 'sms-3.5'];
+            var start_time_offset = 0;
+            messages.forEach(function(message) {
+                project.scheduleMessage({
+                    content: getMessage(message, {}, lang),
+                    to_number: contact.phone_number,
+                    start_time_offset: start_time_offset
+                });
+                start_time_offset +=15;
```
Comment (shejasabin) [inline soil-fertility-trainings/responseHandlers/batch2ResponseHandler.js:20]: Did it work? setting some time in between

Comment (Bahatiroben) [reply soil-fertility-trainings/responseHandlers/batch2ResponseHandler.js:20]: yeah. it worked. with 15 seconds between these, the messages gets delivered in order. @shejasabin

File: ke-legacy/main.js (hunk 1 above)
Comment (DevFojo) [reply ke-legacy/main.js:47]: Okay

**Review by shejasabin — APPROVED** [review]
code looks good

**Review by DevFojo — APPROVED** [review]

**Pull request merged**